src/app/src/components/AboutModal.jsx

```diff
@@ -81,14 +81,14 @@ const AboutModal = ({ isOpen, onClose }) => {
                                     href='https://about.fb.com/news/2020/03/closing-the-gender-data-gap/'
                                     title='Project 17'
                                 />
-                                , two Facebook programs that use data to drive
+                                , two Meta programs that use data to drive
                                 progress on critical humanitarian issues. Data
                                 for Good provides tools built from
                                 privacy-protected data on Facebook’s platform,
```

Comment:
Should this stay "Facebook's platform" or be "Meta's platform"?

Reply:
Meta's program but Facebook's platform - they collected the data on the Facebook application, but the program is run by Meta.